src/Robot-Dobot_MG400/main.py

```diff
@@ -195,10 +203,21 @@ def deinitialize(self):
 
     def configure(self):
 
+        self.movement_height = float(self.movement_height)
         self.set_acceleration_linear(self.acceleration_factor)  # Linear acceleration factor 1-100
         self.set_speed_global(self.global_speed_factor)  # Global speed factor 1-100
         self.set_speed_linear(self.speed_factor)  # Linear speed factor 1-100
-        self._last_xyzr = (None, None, None, None)
+        # self._last_xyzr = (None, None, None, None)
```

Comment:
This line can be cleaned up (together with a couple of old prints)

Alternatively, it could also be placed in init to make sure _last_xyzr is set at all